### PR TITLE
proposed change to fix finish pmutation issue #29

### PR DESCRIPTION
**Pull request by awkay**
See #29

### CODE REVIEW

File: src/main/fulcro/incubator/pessimistic_mutations.cljc [hunk 1]
```diff
@@ -179,13 +179,17 @@
...
+                                         (prim/db->tree (prim/get-query returning) data @state)
+                                         data))
           {::keys [status]} mutation-response-swap
-          hard-error? (= status :hard-error)
-          api-error?  (contains? mutation-response-swap ::mutation-errors)
-          had-error?  (or hard-error? api-error?)]
+          hard-error?                (= status :hard-error)
+          api-error?                 (contains? mutation-response-swap ::mutation-errors)
+          had-error?                 (or hard-error? api-error?)]
+      (log/info mutation-response-swap)
```
Comment (wilkerlucio) [inline src/main/fulcro/incubator/pessimistic_mutations.cljc]: we intend to leave this here?

Comment (awkay) [reply src/main/fulcro/incubator/pessimistic_mutations.cljc]: oops. no.

**Comment by wilkerlucio** [comment]
Just did some trials, working fine to me. 👍

**Comment by levitanong** [comment]
Works with my code as well!

**Pull request closed without merging**